01-node-tutorial/answers/04-names.js

```diff
@@ -0,0 +1,7 @@
+//local
+const secret = 'SUPER SECRET'
+//shared
+const john = 'john'
+const peter = 'peter'
```

Comment:
Challenge question: what's the difference between local and shared? How would we need to change this file to make `secret` shared or public?

Reply:
We would have to include the `secret` variable in the `exports` node module 

`module.exports = {john,peter,secret}`

Reply:
Good, thanks!